zeebe/stream-platform/src/main/java/io/camunda/zeebe/stream/api/scheduling/ProcessingScheduleService.java

```diff
@@ -61,4 +61,33 @@ public interface ProcessingScheduleService extends SimpleProcessingScheduleServi
    *     execution and have no effect.
    */
   ScheduledTask runAtAsync(final long timestamp, final Task task);
+
+  /**
+   * Same as @{@link #runAtFixedRate(Duration, Task)}, but the task is executed on a specific pool
+   * (actor).
+   *
+   * @param delay The delay to wait initially and between each run
+   * @param task The task to execute at the fixed rate
+   * @param pool The pool to execute the task on
+   */
+  void runAtFixedRateOnPool(Duration delay, Task task, AsyncSchedulePool pool);
```

Comment:
:wrench: I wouldn't bother with the `onPool` suffix, as normal method dispatch works just fine :shrug: This is, after all, just a specification of the `runAtFixedRate` call.